Makefile

```diff
@@ -1,7 +1,7 @@
 all: vet staticcheck test
 
 test:
-	go test -covermode=count -coverprofile=coverage.out .
+	go test -race -covermode=atomic -count=1 -coverprofile=coverage.out .
```

Comment:
Could you update the .github/workflows/ci.yml file, too?

Reply:
Good call!